addons/mail/static/tests/chatter_tests.js

```diff
@@ -113,7 +113,9 @@ QUnit.test('list activity widget with no activity', async function (assert) {
         data: this.data,
         arch: '<list><field name="activity_ids" widget="list_activity"/></list>',
         mockRPC: function (route) {
-            assert.step(route);
+            if (new RegExp('dataset/search_read|init_messaging').test(route)) {
```

Comment:
Don't ever use a regex if you can use anything else (such as in this case ideally a strict equality or using `includes`). Same in whole diff.
Regex is usually much slower, and always hard to understand and maintain.

Reply:
Btw what are the extra queries that you are trying to filter out?
I like that this test showed us exactly what was happening, even though its name is completely wrong for this purpose.

Reply:
Ok I saw the extra rpc in enterprise, and my first feeling is that we should not have them (see enterprise PR comment) and this whole diff should be reverted.